test/spec/tags/style.js

```diff
@@ -82,4 +82,21 @@ test('style[inline]: background image', async assert => {
   const output = template({}, escape)
   assert.truthy(output.includes('url(data:image/jpg;charset=utf-8;base64'))
   assert.truthy(output.includes('FFFFABRRRQAUUUUAFFFFABRRRQAUUUUAFFFFAH//2Q=='))
-})
\ No newline at end of file
+})
+
+test('style[colors]: custom colors', async assert => {
+  var { template } = await compile(`<style>.button { color: red; }</style>`, { styles: { colors: { red: '#FF6347' } } })
+  assert.deepEqual(template({}, escape), '<style>.button { color: #FF6347; }</style>')
+})
+
+test('style[colors]: mixed colors', async assert => {
+  var { template } = await compile(`<style>.button { color: red; } .blue.button { color: myBeautifulBlue; } .yellow.button { color: yellow }</style>`, { 
+    styles: { 
+      colors: { 
+        red: '#FF6347',
```

Comment:
dodajmy jeszcze osobne zadanko na sprawdzenie, czy przekazana wartość koloru jest poprawna

Reply:
Nie jestem przekonany czy powinniśmy dodawać taką walidacje.
Użytkownik powinien sam wiedzieć co chce osiągnać. Jeśli podaje śmieci na wejściu to taki sam rezultat otrzyma na końcu.

Reply:
chodzi raczej o drobne pomyłki, jak np. literówki lub przekazanie pustej zmiennej